jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -875,3 +875,11 @@ def _fft_rng_factory(dtype):
                                np.array([0, 0xFFFFFFFF], dtype=np.uint32),
                                np.array([0xFFFFFFFF, 0xFFFFFFFF], dtype=np.uint32)])
 )
+
+type_promotion = tuple(
```

Comment:
This is not really a primitive harness. I am thinking that perhaps all that is needed is to replace the `jtu.cases_from_list` with `tuple`?

Reply:
That's fair; I wasn't 100% sure about making it a "primitive harness" since it is really testing something at a higher level. I made the relevant change in `c104889`. Please use that second commit message when merging, then :)